src/Frontend/Themes/Bootstrap4/Modules/Profiles/Layout/Widgets/LoginBox.html.twig

```diff
@@ -8,34 +8,31 @@
     {{ 'msg.ProfilesLoggedInAs'|trans|format(profile.display_name, profile.url.dashboard)|raw }} -
     <a href="{{ profile.url.settings }}">{{ 'lbl.Settings'|trans|ucfirst }}</a>
   {% else %}
-    <header>
-      <h3>{{ 'lbl.Login'|trans|ucfirst }}</h3>
-    </header>
+    <h3>{{ 'lbl.Login'|trans|ucfirst }}</h3>
     {% form login %}
       <fieldset>
-        <div class="form-group row{% if txtEmailError %} has-danger{% endif %}">
-          <label class="form-control-label col-sm-2" for="email">{{ 'lbl.Email'|trans|ucfirst }}<abbr title="{{ 'lbl.RequiredField'|trans|ucfirst }}">*</abbr></label>
-          <div class="col-sm-6">
-            {% form_field_error email %}{% form_field email %}
-          </div>
+        <div class="form-group{% if txtEmailError %} has-danger{% endif %}">
+          <label class="form-control-label" for="email">{{ 'lbl.Email'|trans|ucfirst }}<abbr title="{{ 'lbl.RequiredField'|trans|ucfirst }}">*</abbr></label>
+          {% form_field email %}
+          {% form_field_error email %}
         </div>
-        <div class="form-group row{% if txtPasswordError %} has-danger{% endif %}">
-          <label class="form-control-label col-sm-2" for="password">{{ 'lbl.Password'|trans|ucfirst }}<abbr title="{{ 'lbl.RequiredField'|trans|ucfirst }}">*</abbr></label>
-          <div class="col-sm-6">
-            {% form_field_error password %}{% form_field password %}
-          </div>
+        <div class="form-group{% if txtPasswordError %} has-danger{% endif %}">
+          <label class="form-control-label" for="password">{{ 'lbl.Password'|trans|ucfirst }}<abbr title="{{ 'lbl.RequiredField'|trans|ucfirst }}">*</abbr></label>
+          {% form_field password %}
+          {% form_field_error password %}
+          <small><a href="{{ geturlforblock('Profiles', 'ForgotPassword') }}" title="{{ 'msg.ForgotPassword'|trans }}">{{ 'msg.ForgotPassword'|trans }}</a></small>
         </div>
-        <div class="form-group row">
-          <div class="offset-sm-2 col-sm-6">
-          <label for="remember">{% form_field remember %} {{ 'lbl.RememberMe'|trans|ucfirst }}</label>
-            {% form_field_error remember %}
+        <div class="form-group">
+          <div class="custom-control custom-checkbox">
+            {% form_field remember %}
```

Comment:
Heeft deze wel de juiste class voor die custom chechbox? want dat zit in fork fork